### ISSUE TITLE
Lyric language problem

### ISSUE THREAD

**Issue by devindang** [digimezzo/dopamine-windows#965]
When the software language is set to Chinese, the lyrics obtained are also Chinese. Even English songs can't get English lyrics.

**Comment by digimezzo** [comment]
Thanks for reporting this. I've added the google translate translation to your post. That'll make it easier for me to understand :)

**Comment by Magentaize** [comment]
What lyric provider are you using?

**Comment by devindang** [comment]
> Thanks for reporting this. I've added the google translate translation to your post. That'll make it easier for me to understand :)

Thank you

**Comment by devindang** [comment]
> What lyric provider are you using?

Netease music . The lyrics of Chinese songs can't be found via other providers.

**Comment by devindang** [comment]
> What lyric provider are you using?

I prefer it shows both Chinese and English lyrics. I hope you can consider it.

**Comment by Magentaize** [comment]
> > What lyric provider are you using?
> 
> I prefer it shows both Chinese and English lyrics. I hope you can consider it.

Yes, the Netease provider supports lyric in two languages at the same time and it's controlled by software language. I'll make a change into it.
